charts/thoras/templates/crd/aiscaletarget.yaml

```diff
@@ -105,8 +105,28 @@ spec:
                         - type: integer
                         - type: string
                       x-kubernetes-int-or-string: true
-                    forecast_buffer_percentage:
-                      x-kubernetes-int-or-string: true
+                reasoning:
+                  type: object
+                  required:
+                    - metrics
+                  properties:
+                    metrics:
+                      type: array
+                      items:
+                        type: object
+                        required:
+                            - type
```

Comment:
bikeshedding time: what about `"provider"` instead of `type`? Seems more accurate